Servers/UI/OJS.Servers.Ui/ClientApp/src/pages/administration-new/submission-types/AdministrationSubmissionTypesPage.tsx

```diff
@@ -22,7 +28,14 @@ const AdministrationSubmissionTypesPage = () => {
     // eslint-disable-next-line max-len
     const [ queryParams, setQueryParams ] = useState<IGetAllAdminParams>(applyDefaultFilterToQueryString('', defaultSorterToAdd, searchParams));
 
-    const { refetch, data: submissionTypesData, isLoading: isGettingData, error } = useGetAllSubmissionTypesQuery(queryParams);
+    const {
```

Comment:
When `refetching` data and we want to display a `<SpinningLoader />` while the data is being loaded, it is better to use the `isFetching` flag, not the `isLoading` flag. The `isLoading` flag is true only for the initial request of the data, it remains `false` for all subsequent requests. In contrast, `isFetching` is true for both initial and subsequent requests, e.g., when `refetching` data. This ensures the loader is displayed during all data loading operations.

Reply:
Implemented this